manage/ships/repair.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+require '../layout/header.php';
+
+use Service\Container;
+
+$id = isset($_GET['id']) ? $_GET['id'] : null;
+
+$container = new Container($configuration);
+$shipLoader = $container->getShipLoader();
+// get our ship
+$ship = $shipLoader->findOneById($id);
+$ship->repairShip(); // updated currenthealth on the ship obj
```

Comment:
This can now be removed. See note in PdoShipStorage

Reply:
repairShip Removed